src/status_im2/common/standard_authentication/standard_auth/view.cljs

```diff
@@ -43,14 +43,16 @@
                         :shell?   blur?
                         :content  (fn []
                                     [enter-password/view
-                                     {:customization-color customization-color
+                                     {:profile-color       profile-color
+                                      :customization-color customization-color
                                       :on-enter-password   on-enter-password
                                       :button-icon-left    auth-button-icon-left
                                       :button-label        auth-button-label}])}]))))))
 
 (defn- view-internal
   [_]
-  (let [reset-slider? (reagent/atom false)
+  (let [profile-color (rf/sub [:profile/customization-color])
```

Comment:
I don't think we should bind a sub to standard auth here. Imo it's better to keep this free and pass it as a prop from the screen where it's being used 👍 that's what I did on other wallet screens. Then it can be community color, account color or user color/ who knows what else might be needed eventually

Reply:
You are right. The customization color of the slider button would reflect the context the user is in and we will need a sub not in the `standard-auth` view .

But, in the `enter password` bottom sheet, the customization color of the `confirm` button should be from the currently logged-in user's profile color.

<img width="600"  src="https://github.com/status-im/status-mobile/assets/19339952/1c9add37-f73d-47d0-99a6-7f72fed3d298">

I will remove the sub and customization color passed to the `enter-password` component/bottom-sheet. 👍